rpc/apis.go

```diff
@@ -16,7 +14,7 @@ const PersonalNamespace = "personal"
 const NetNamespace = "net"
 
 // GetRPCAPIs returns the list of all APIs
-func GetRPCAPIs(cliCtx context.CLIContext, key emintcrypto.PrivKeySecp256k1) []rpc.API {
+func GetRPCAPIs(cliCtx context.CLIContext, keys []emintcrypto.PrivKeySecp256k1) []rpc.API {
```

Comment:
nit, can we change it to ellipsis everywhere?
```suggestion
func GetRPCAPIs(cliCtx context.CLIContext, keys ...emintcrypto.PrivKeySecp256k1) []rpc.API {
```

Reply:
will leave as array for now!